sklearn/mixture/_gaussian_mixture_ic.py

```diff
@@ -231,15 +235,18 @@ class GaussianMixtureIC(ClusterMixin, BaseEstimator):
     Notes
     -----
     This algorithm was strongly inspired by mclust, a clustering package in R
+    https://sites.stat.washington.edu/mclust/
 
     References
     ----------
     .. [1] Jeffrey D. Banfield and Adrian E. Raftery. Model-based gaussian and
        non-gaussian clustering. Biometrics, 49:803–821, 1993.
+       https://doi.org/10.2307/2532201
 
     .. [2] Abhijit Dasgupta and Adrian E. Raftery. Detecting features in
        spatial point processes with clutter via model-based clustering.
        Journal of the American Statistical Association, 93(441):294–302, 1998.
+       https://doi.org/10.2307/2669625
```

Comment:
You can make use of the [cross-referencing syntax](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html).

I've added the reference of `mclust` and its paper DOI.
```suggestion
    Notes
    -----
    This algorithm was strongly inspired by mclust [3]_,
    a clustering package for R.

    References
    ----------
    .. [1] Jeffrey D. Banfield and Adrian E. Raftery. Model-based gaussian and
       non-gaussian clustering. Biometrics, 49:803–821, 1993.
       <https://doi.org/10.2307/2532201>_

    .. [2] Abhijit Dasgupta and Adrian E. Raftery. Detecting features in
       spatial point processes with clutter via model-based clustering.
       Journal of the American Statistical Association, 93(441):294–302, 1998.
       <https://doi.org/10.2307/2669625>_

    .. [3] Scrucca, L., Fop, M., Murphy, T. B., & Raftery, A. E. (2016).
       mclust 5: Clustering, Classification and Density Estimation Using 
       Gaussian Finite Mixture Models. The R journal, 8(1), 289–317. 
       <https://doi.org/10.32614/RJ-2016-021>_
       
```